warehouse/templates/base.html

```diff
@@ -12,6 +12,33 @@
  # limitations under the License.
 -#}
 
+{% macro input_field_label(user_friendly_name, formInstance) -%}
+<label for="{{formInstance.name}}" class="form-group__label">
+  {% trans form_name=user_friendly_name %}{{ form_name }}{% endtrans %}
+  {% if formInstance.flags.required %}
+  <span class="form-group__required">{% trans %}(required){% endtrans %}</span>
+  {% endif %}
+</label>
+{% endmacro %}
+
+{% macro input_field_errors(name, formInstance) -%}
+<div id="{{name}}-errors">
+  {% if formInstance.errors %}
+  <ul class="form-errors" role="alert">
+    {% for error in formInstance.errors %}
+    <li>{{ error }}</li>
+    {% endfor %}
+  </ul>
+  {% endif %}
+</div>
+{% endmacro %}
+
+{% macro input_field(user_friendly_name, name, formInstance, placeholder, autocomplete="", required=True, data_action="", data_target="") -%}
```

Comment:
```suggestion
{% macro input_field(name, user_friendly_name, form_instance, placeholder, autocomplete="", required=True, data_action="", data_target="") -%}
```
Can you reorder/rename the arguments here (and elsewhere) to match?